elements/pfe-card/src/pfe-card.scss

```diff
@@ -58,139 +60,171 @@ $LOCAL: card;
   overflow: hidden;
 
   // Base colors
-  background-color: #{pfe-local(BackgroundColor)};
+  background-color:    #{pfe-local(BackgroundColor)};
   background-position: #{pfe-local(BackgroundPosition)};
-  color: #{pfe-radio(color, text)};
+  color:               #{pfe-radio(color, text)} !important;
 }
 
 // @TODO Deprecate "dark" and "light" in favor of "darker" and "lighter"
-@each $color in (dark, darker, darkest, light, lighter, lightest, complement, accent) {
+@each $color in (darker, darkest, lighter, lightest, complement, accent) {
   $label: $color;
-  // Add backward support for dark and light
-  @if $color == "dark" or $color == "light" {
-    $color: "#{$color}er";
-  }
   // Update variable values to match provided attribute
-  // @TODO deprecate [color] in favor of [pfe-color]
-  :host([color="#{$label}"]),
   :host([pfe-color="#{$label}"]) {
     --pfe-card--BackgroundColor:                  #{pfe-color(surface--#{$color})};
     --pfe-broadcasted--color--text:               #{pfe-color(surface--#{$color}--text)};
     --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--#{$color}--link)};
     --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--#{$color}--link--visited)};
     --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--#{$color}--link--hover)};
     --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--#{$color}--link--focus)};
-
   }
 }
 
-// @TODO deprecate [size] in favor of [pfe-size]
-:host([size="small"]),
 :host([pfe-size="small"]) {
   --pfe-card--PaddingTop:     #{pfe-var(container-spacer)};
   --pfe-card--PaddingRight:   #{pfe-var(container-spacer)};
   --pfe-card--PaddingBottom:  #{pfe-var(container-spacer)};
   --pfe-card--PaddingLeft:    #{pfe-var(container-spacer)};
 }
 
-::slotted([pfe-overflow~="top"]) {
-  z-index: 1;
-  margin-top: -2rem; //IE11 fallback
-  margin-top: calc(-1 * #{pfe-local(PaddingTop)}) !important;
-  :host([has_header]) & {
-    padding-top: #{pfe-local(spacing)};
-  }
-}
+// Targets the wrappers in the shadow DOM
+.pfe-card {
+  &__header,
+  &__body,
+  &__footer {
+    ::slotted([pfe-overflow~="top"]) {
+      z-index: 1;
+      margin-top: -2rem; //IE11 fallback
+      margin-top: calc(-1 * #{pfe-local(PaddingTop)}) !important;
+      :host([has_header]) & {
+        padding-top: #{pfe-local(spacing)};
+      }
+    }
 
-::slotted([pfe-overflow~="right"]) {
-  margin-right: -2rem; //IE11 fallback
-  margin-right: calc(-1 * #{pfe-local(PaddingRight)});
-}
+    ::slotted([pfe-overflow~="right"]) {
```

Comment:
@markcaron are you still working on cleanup of this branch? Asking because I still see overflow attributes here

Reply:
AFAIK, these overflow attributes exist in master already. When I reverted the img distortion changes I made in this branch, I put back what existed in master.